### PR TITLE
Minor tweaks, get up and running with the example more easily

### PR DESCRIPTION
**Pull request by austinvhuang**
- Added clarifications for getting the example working with stack
- Included example program under examples/
- Bumped LTS to 8.5
- README clarifications & cleanup

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+stack exec hylide examples/Example.hs
```
Comment (sleexyz) [inline run.sh]: Is this file necessary? It's not referenced in the README. 

Omit this file and you're golden :)

**Comment by austinvhuang** [comment]
@sleexyz done
